src/components/stepforms/widgets/DateComponents/DateRangeInput.vue

```diff
@@ -12,11 +12,13 @@
           class="widget-date-input__editor-side"
           :availableVariables="availableVariables"
           :selectedVariables="selectedVariables"
+          :enableRelativeDate="enableRelativeDate"
           @selectCustomVariable="editCustomVariable"
           @input="selectVariable"
         />
         <div class="widget-date-input__editor-content" v-show="isCustom" ref="custom-editor">
           <Tabs
+            v-if="enableRelativeDate"
```

Comment:
I see below that `RangeCalendar` is a child of this tabs component. So if we stop showing tabs this component will never show a `RangeCalendar` but only the `CustomVariableList` ?

Reply:
Nop it's not a child, it's depedns only on selectedTab state property, if the component is display or not, so it has no impact to hide the tabs component, it will just remove navigation